perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_group_resource_attribute_def_virt_unixGID.java

```diff
@@ -37,8 +37,6 @@ public void checkAttributeSemantics(PerunSessionImpl sess, Group group, Resource
 		gidAttribute.setValue(attribute.getValue());
 		try {
 			sess.getPerunBl().getAttributesManagerBl().forceCheckAttributeSemantics(sess, group, gidAttribute);
```

Comment:
You are checking both syntax and semantics in method checkAttributeSemantics in module virt_googleGroupName, but here you are checking just forceCheckAttributeSemantics. Shouldn't it be implemented the same way for all virt modules? What do you think @stavamichal ?

Reply:
We should preserve old behavior here, so you should add method forceCheckAttributeSyntax here (it does not exist now so you need to create this method).